docs/deployment/recovery/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Recovering from failure"
+---
+
+If you run into an error when operating wasmCloud, the first step is to diagnose the problem. Typically, this will mean checking three sources of information:
+
+* Logs for the [wasmCloud host](/docs/concepts/hosts.mdx) process
+* Logs for the [wasmCloud Application Deployment Manager (`wadm`)](/docs/ecosystem/wadm/index.md) process
+* Activity on [NATS](/docs/ecosystem/nats/index.mdx)
+
+## Troubleshooting on Kubernetes
+
+When deploying [wasmCloud on Kubernetes](/docs/deployment/k8s/index.mdx), you can access logs for wasmCloud hosts and `wadm` with `kubectl`. 
+
+To check logs for `wadm`:
+
+```shell
+kubectl logs -f -l app.kubernetes.io/instance=wadm -c wadm
+```
+
+To check logs for a wasmCloud host:
+
+```shell
+kubectl logs -f -l app.kubernetes.io/instance=my-wasmcloud-cluster -c wasmcloud-host
```

Comment:
it might be worth adding a note about how to determine what the value of `my-wasmcloud-cluster` should be below this, because there's a good chance the value other folks have for this doesn't match this.